providers/cncf/kubernetes/tests/unit/cncf/kubernetes/utils/test_pod_manager.py

```diff
@@ -103,8 +73,11 @@ def test_read_pod_logs_successfully_returns_logs(self):
         assert isinstance(logs, PodLogsConsumer)
         assert logs.response == mock.sentinel.logs
 
-    @mock.patch("airflow.providers.cncf.kubernetes.utils.pod_manager.get_retry_after_seconds", wait_none)
-    def test_read_pod_logs_retries_successfully(self):
+    @mock.patch(
```

Comment:
Nit: Very many similar/same patches, would it make sense to make this a general fixture to potentially auto-use?